src/ndn/transport/udp_face.py

```diff
@@ -0,0 +1,85 @@
+# -----------------------------------------------------------------------------
+# Copyright (C) 2019-2020 The python-ndn authors
+#
+# This file is part of python-ndn.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+import asyncio as aio
+import logging
+from .face import Face
+from ..encoding.tlv_var import parse_tl_num
+
+
+class UdpFace(Face):
+    host: str = '127.0.0.1'
+    port: int = 6363
+
+    def __init__(self, host: str = '', port: int = 0):
```

Comment:
Declare the defaults 127.0.0.1 and 6363 on this line.
Get rid of the `if` statements.
The member declarations in above three lines are also unnecessary.